wd_sched.c

```diff
@@ -117,8 +120,10 @@ static struct sched_ctx_region *sched_get_ctx_range(struct wd_sched_ctx *ctx,
 
 	/* If the key->numa_id is not exist, we should scan for a region */
 	for (numa_id = 0; numa_id < ctx->numa_num; numa_id++) {
-		if (sched_info[numa_id].ctx_region[key->mode][key->type].valid)
-			return &sched_info[numa_id].ctx_region[key->mode][key->type];
+		for (type = 0; type < ctx->type_num; type++) {
+			if (sched_info[numa_id].ctx_region[key->mode][type].valid)
+				return &sched_info[numa_id].ctx_region[key->mode][type];
+		}
```

Comment:
Why do you change from **key->type** to **type**?

Reply:
In line 119, the scene where the key has been filled with parameters has been processed. 
In fact, the parameters in the key are all default, and a valid region can only be found by querying

Reply:
Your assumption is that there's only one **type** in the ctx_region. If there're multiple **type**s in the ctx_region, what will happen?

Reply:
If there are multiple types in ctx_region, when there are input parameters, the region can be obtained directly through the parameters. If there are no input parameters, a region in the nearest numa field is returned for the user to use. If the types do not match, an error will be reported in the uadk driver